tests/baselines/reference/commentsArgumentsOfCallExpression2.js

```diff
@@ -18,6 +18,6 @@ foo(/*c3*/ function () { }, /*d2*/ function () { }, /*e2*/ a + b);
 foo(/*c3*/ function () { }, /*d3*/ function () { }, /*e3*/ (a + b));
 foo(
 /*c4*/ function () { }, 

```

Comment:
Well, the original issue seems more specific - https://github.com/Microsoft/TypeScript/issues/4629

Fixing all trailing spaces in all scenarios seems like much larger effort than the tag "easy" suggests... ;)